packages/shared/src/utils/config/config.ts

```diff
@@ -43,10 +43,10 @@ export const isTestLink = () => {
 };
 
 export const getAppId = () => {
-    let app_id = null;
+    let app_id: number | string | null = null;
     const user_app_id = ''; // you can insert Application ID of your registered application here
     const config_app_id = window.localStorage.getItem('config.app_id');
-    const current_domain = getCurrentProductionDomain();
+    const current_domain: string = getCurrentProductionDomain() || '';
     const platform = new URLSearchParams(window.location.search).get('platform');
```

Comment:
I think we will not need to define a type here, if we define return type for the function. correct me please